sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/pending/PendingTransactionDb.kt

```diff
@@ -24,11 +27,67 @@ import kotlinx.coroutines.flow.Flow
     entities = [
         PendingTransactionEntity::class
     ],
-    version = 1,
+    version = 2,
     exportSchema = true
 )
 abstract class PendingTransactionDb : RoomDatabase() {
     abstract fun pendingTransactionDao(): PendingTransactionDao
+
+    companion object {
+
+        /*
+         * Non-automatic migration required because to_address became nullable.
+         */
+        internal val MIGRATION_1_2 = object : Migration(1, 2) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL(
+                    """
+                 ALTER TABLE pending_transactions RENAME TO pending_transactions_old;
+                 CREATE TABLE pending_transactions(
+                     id                         INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+                     toAddress                  TEXT,
+                     toInternalAccountIndex     INTEGER,
+                     accountIndex               INTEGER NOT NULL,
```

Comment:
Could this be named something like `sentFromAccountIndex` or something of the sort?

Reply:
I had preserved the original column names, which are derived from the original Java object field names.  This can be renamed easily enough.